lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1206,6 +1206,7 @@ def _on_move(self, event):
             self.elev = art3d.norm_angle(self.elev - (dy/h)*180)
             self.azim = art3d.norm_angle(self.azim - (dx/w)*180)
             self.get_proj()
+            self.stale = True
```

Comment:
Yeah, this makes sense. Don't forget to do it for the zoom action below.

Reply:
Zoom doesn’t need it because the calls to set limits already take care of setting stale.